File: Java /Gate-two/level-4/testingArraystask.py
from unittest import TestCase
from arrays_Task


class TestForArrays(TestCase):
    def test_for_numbers_in_an_array(self):
        size = 5
        actual = arraysTask.numbers_in_an_array{10,11,12,5}
        expected = {10,11,12,-1,-1}
        assertEqual(actual, expected)

File: Java /Gate-two/level-4/testingArraystask.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertArrayEquals;


public class testingArraystask{


        @Test
    public void arraysTask(){
    int size = 5; 

    int actual = Daythree.Arraystask(10,11,12,5);

    int expected = (10,11,12,-1,-1);
    assertArrayEquals(actual, expected)
    }
}
